sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/StoreResponseTest.java

```diff
@@ -19,7 +19,7 @@ public void stringContent() {
         headerMap.put("key1", "value1");
         headerMap.put("key2", "value2");
 
-        StoreResponse sp = new StoreResponse(200, new ArrayList<>(headerMap.entrySet()), getUTF8BytesOrNull(content));
+        StoreResponse sp = new StoreResponse(200,headerMap, getUTF8BytesOrNull(content));
```

Comment:
code style space after comma
```suggestion
        StoreResponse sp = new StoreResponse(200, headerMap, getUTF8BytesOrNull(content));
```